xbmc/epg/GUIEPGGridContainer.cpp

```diff
@@ -114,11 +114,11 @@ CGUIEPGGridContainer::CGUIEPGGridContainer(const CGUIEPGGridContainer &other)
   m_focusedProgrammeLayouts = other.m_focusedProgrammeLayouts;
   m_programmeLayouts        = other.m_programmeLayouts;
   m_rulerLayouts            = other.m_rulerLayouts;
-  m_channelLayout           = other.m_channelLayout;
-  m_focusedChannelLayout    = other.m_focusedChannelLayout;
-  m_programmeLayout         = other.m_programmeLayout;
-  m_focusedProgrammeLayout  = other.m_focusedProgrammeLayout;
-  m_rulerLayout             = other.m_rulerLayout;
+  m_channelLayout           = NULL;
+  m_focusedChannelLayout    = NULL;
+  m_programmeLayout         = NULL;
+  m_focusedProgrammeLayout  = NULL;
+  m_rulerLayout             = NULL;
```

Comment:
This changes behaviour, is it tested to not break anything? Gut feeling says we should do a deep copy but I leave a final decision to the pvr team. Thanks for highlighting the issue even if we disagree about the fix in the end.